pkg/background/mutate/mutate.go

```diff
@@ -213,17 +213,25 @@ func (c *mutateExistingController) ProcessUR(ur *kyvernov1beta1.UpdateRequest) e
 	return updateURStatus(c.statusControl, *ur, err)
 }
 
-func (c *mutateExistingController) getPolicy(key string) (kyvernov1.PolicyInterface, error) {
-	pNamespace, pName, err := cache.SplitMetaNamespaceKey(key)
+func (c *mutateExistingController) getPolicy(ur *kyvernov1beta1.UpdateRequest) (policy kyvernov1.PolicyInterface, err error) {
+	pNamespace, pName, err := cache.SplitMetaNamespaceKey(ur.Spec.Policy)
 	if err != nil {
 		return nil, err
 	}
 
 	if pNamespace != "" {
-		return c.npolicyLister.Policies(pNamespace).Get(pName)
+		policy, err = c.npolicyLister.Policies(pNamespace).Get(pName)
 	}
 
-	return c.policyLister.Get(pName)
+	policy, err = c.policyLister.Get(pName)
```

Comment:
I guess this should be in an `else`?

Reply:
Right, I fixed it locally. I'm looking at a bunch of validate tests failures, will push it later.